src/PodActions.jsx

```diff
@@ -141,7 +141,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "unpause", pod.Id, {})
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to resume pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-062427-2b92d0d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)